.github/workflows/publish.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish
+on:
+  release:
+    types: [released]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: curl -L ${{ github.event.release.tarball_url }} > source.tar.gz
+      - uses: CheckerNetwork/publish-zinnia-module-action@v0
+        id: publish
+        with:
+          source: source.tar.gz
+          w3up-private-key: ${{ secrets.W3UP_PRIVATE_KEY }}
+          w3up-proof: ${{ secrets.W3UP_PROOF }}
+          w3name-private-key: ${{ secrets.W3NAME_PRIVATE_KEY }}
+          w3name-revision: ${{ secrets.W3NAME_REVISION }}
```

Comment:
We need to use a unique w3name identity for each subnet. Can you please confirm you created new w3name credentials for the Arweave checker?

Reply:
We do not have w3name created for arweave checker. I believe we should do that using web3.storage console? I'm struggling trying to find access credentials for web3.storage in 1Password.

Reply:
See this script for generating credentials: https://github.com/CheckerNetwork/publish-zinnia-module-action/blob/2e0211ca6021228bca6c33b0aa55a73260518f68/scripts/w3name.js

Reply:
w3name is not managed through web3storage

Reply:
Thanks @juliangruber for linking me to the script, I've managed to create w3name credentials using it. But there eems like there's an additional credentials needed for [w3up](https://docs.storacha.network/w3up-client/). 

As I understand we would need to provision a "Space" by logging in using some email. Once that's done we would get `did` credential that we could use to upload our package to IPFS?